source/pathEq.js

```diff
@@ -14,7 +14,7 @@ import path from './path.js';
  * @typedefn Idx = String | Int | Symbol
  * @sig a -> [Idx] -> {a} -> Boolean
  * @param {*} val The value to compare the nested property with
- * @param {Array} path The path of the nested property to use
+ * @param {Array} pathAr The path of the nested property to use
```

Comment:
Not sure why this was updated, just `path` probably preferable? But I'm not sure what you were going for here with `pathAr`?

Reply:
I changed it when i changed the parameter name in the function, but you're right, doesn't make sense to change the docs. 
However when trying to retrace my thought process, I discovered that there are a bunch of inconsistencies across the "path" related functions, both in docs and in fn param names. For example pathOr both documents path as "p", and compared to propOr, the default/or value is documented as "d" in one and "val" in the other. I think that goes beyond this PR though, so I might submit another later on to try and fix those.